test/common/http/dispatcher_test.cc

```diff
@@ -773,17 +773,29 @@ TEST_F(DispatcherTest, MultipleStreams) {
   ASSERT_EQ(cc.on_complete_calls, 1);
 }
 
-TEST_F(DispatcherTest, EnvoyLocalReply) {
+TEST_F(DispatcherTest, EnvoyLocalReplyNotAnError) {
```

Comment:
This is the suite where you are going to add more test? Or are you creating a test suite for the filter? Probably both, right?

Reply:
Will add both.

Reply:
Per offline discussion it sounds like you will add in subsequent PRs. Do you mind opening issue for that?

Reply:
Sure - https://github.com/envoyproxy/envoy-mobile/issues/1207